qiskit/pulse/commands/acquire.py

```diff
@@ -113,52 +118,89 @@ class AcquireInstruction(Instruction):
+                or isinstance(reg_slot, list):
+            warnings.warn('AcquireInstruction on multiple qubits, multiple memory slots '
+                          'and multiple reg slots has been deprecated. Parameter "mem_slots" '
+                          'has been replaced by "mem_slot" and "reg_slots" has been '
+                          'replaced by "reg_slot"', DeprecationWarning, 3)
+
+        if not isinstance(acquire, list):
+            acquire = [acquire]
 
-        if isinstance(acquires[0], Qubit):
+        if isinstance(acquire[0], Qubit):
             raise PulseError("AcquireInstruction can not be instantiated with Qubits, "
                              "which are deprecated.")
 
-        if not (mem_slots or reg_slots):
+        if mem_slot and not isinstance(mem_slot, list):
+            mem_slot = [mem_slot]
+        elif mem_slots:
+            mem_slot = mem_slots
+
+        if reg_slot:
+            reg_slot = [reg_slot]
+        elif reg_slots and not isinstance(reg_slots, list):
+            reg_slot = [reg_slots]
+        else:
+            reg_slot = reg_slots
+
+        if not (mem_slot or reg_slot):
             raise PulseError('Neither memoryslots or registers were supplied')
 
-        if mem_slots:
-            if isinstance(mem_slots, MemorySlot):
-                mem_slots = [mem_slots]
-            elif len(acquires) != len(mem_slots):
-                raise PulseError("#mem_slots must be equals to #acquires")
+        if mem_slot and len(acquire) != len(mem_slot):
+            raise PulseError("#mem_slots must be equals to #acquires")
 
-        if reg_slots:
-            if isinstance(reg_slots, RegisterSlot):
-                reg_slots = [reg_slots]
-            if len(acquires) != len(reg_slots):
+        if reg_slot:
+            if len(acquire) != len(reg_slot):
                 raise PulseError("#reg_slots must be equals to #acquires")
         else:
-            reg_slots = []
+            reg_slot = []
+
+        super().__init__(command, *acquire, *mem_slot, *reg_slot, name=name)
 
-        super().__init__(command, *acquires, *mem_slots, *reg_slots, name=name)
+        self._acquires = acquire
+        self._mem_slots = mem_slot
+        self._reg_slots = reg_slot
 
-        self._acquires = acquires
-        self._mem_slots = mem_slots
-        self._reg_slots = reg_slots
+    @property
+    def acquire(self):
+        """Acquire channel to be acquired on."""
+        return self._acquires[0] if self._acquires else None
+
+    @property
+    def mem_slot(self):
+        """MemorySlot."""
+        return self._mem_slots[0] if self._mem_slots else None
+
+    @property
+    def reg_slot(self):
+        """RegisterSlot."""
+        return self._reg_slots[0] if self._reg_slots else None
 
     @property
     def acquires(self):
         """Acquire channels to be acquired on."""
+        warnings.warn('"acquires" is deprecated and being replaced by "acquire"',
```

Comment:
We can remove this warning for now (or suppress warnings in the scheduler)

Reply:
@ 91e03ef